src/data/queries/staticPathResources.ts

```diff
@@ -56,11 +60,23 @@ export const formatter: QueryFormatter<
   JsonApiResourceWithPathAndFieldAdmin[],
   StaticPathResource[]
 > = (resources: JsonApiResourceWithPathAndFieldAdmin[]) => {
-  return resources.map((resource) => ({
-    path: resource.path.alias,
+  const filteredResources = resources.filter((resource) => {
+    // Filter out resources that don't have a path
+    if (!resource.path) {
+      return false
+    }
+
+    // Filter out resources that don't have an alias
+    if (!resource.path.alias) {
+      return false
+    }
+    return true
+  })
```

Comment:
Can we simplify it like this?
```suggestion
  // Filter out resources that don't have a path or path alias
  const filteredResources = resources.filter((resource) => Boolean(resource.path?.alias))
```